magnum_capi_helm/driver.py

```diff
@@ -691,7 +691,7 @@ def _storageclass_definitions(self, context, cluster):
 
         for volume_type in volume_types:
             storage_class = {
-                "name": volume_type,
+                "name": driver_utils.sanitized_name(volume_type),
```

Comment:
Can you tweak a test to make sure this does the correct thing please?

Reply:
yep, I tweaked the test, just forgot to push it 😅